dcos_openvpn/cert.py

```diff
@@ -28,4 +36,13 @@ def output(name):
 
 
 def remove(name):
-    return os.remove(path(name))
+    subprocess.check_call(
+        '$ZKCLI --run-once "rm $ZKPATH/{0}" $ZKURL'.format(
+            os.path.relpath(key_path(name),
+                os.environ.get("CONFIG_LOCATION"))), shell=True)
+    p = subprocess.Popen(
```

Comment:
Is there a way to move to one of the `check_*` calls for this instead of raw Popen? Maybe put this into the local bash script instead?